interface.lisp

```diff
@@ -6,10 +6,12 @@
 
 (in-package #:org.tymoonnext.dissect)
 
-(declaim (ftype (function () list) stack restarts)
-         (notinline stack restarts))
+(declaim (ftype (function () list) stack)
+         (notinline stack))
 (defun stack ())
 
+(declaim (ftype (function (&optional (or null condition)) list) restarts)
+         (notinline restarts))
 (defun restarts ())
```

Comment:
This definition is not consistent with the declaim

Reply:
Also missing an update for the documentation to explain the parameter

Reply:
[Fixed](3954a9bcb0a0431b77763f2a46775ad51ef9751c) and [fixed](5274b07955c7850f7573436a313e680d74cb64c1)!